tests/e2e/holodeck.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: holodeck.nvidia.com/v1alpha1
+kind: Environment
+metadata:
+  name: HOLODECK_NAME
+  description: "Devel infra environment"
+spec:
+  provider: aws
+  auth:
+    keyName: cnt-ci
+    privateKey: HOLODECK_PRIVATE_KEY
+  instance:
+    type: g4dn.xlarge
+    region: us-west-1
+    ingressIpRanges:
+      - 0.0.0.0/0
```

Comment:
Out of scope for this PR: Do we have logic to restrict the IPs to a specific IP?

Reply:
we have, but GitHub actions have a very broad range of IP's, this will be solved once we move to Self-hosted runner , where the IP range is static